### PR TITLE
week10 codes uploades

### PR DESCRIPTION
**Pull request by fifo36**

### CODE REVIEW

File: week10/17. Letter Combinations of a Phone Number.cpp [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        digitLetters[7] = "pqrs";
+        digitLetters[8] = "tuv";
+        digitLetters[9] = "wxyz";
+
+        vector<string> res;
+        string cur = "";
+        if(digits.size() == 0) return {};
+
+        getString(0, cur, digits, res);
+        cout<<res.size()<<endl;
```
Comment (Sadman007) [inline week10/17. Letter Combinations of a Phone Number.cpp:36]: Please do not keep garbage snippets in your final submissions. It looks very odd!

**Review by Sadman007 — APPROVED** [review]
LGTM

File: week10/17. Letter Combinations of a Phone Number.cpp (hunk 1 above)
Comment (fifo36) [reply week10/17. Letter Combinations of a Phone Number.cpp:36]: It has remained unintentionally.

**Comment by fifo36** [comment]
done

**Pull request closed without merging**